src/components/utility/EmoteCard.vue

```diff
@@ -253,6 +253,19 @@ const openContext = (ev: MouseEvent) => {
 
 			default:
 				break;
+			case "copy-emote-link":
+				navigator.clipboard.writeText(`https://7tv/emotes/${props.emote.id}`).then(() => {
+					/* clipboard successfully set */
+					modal.open("copy-emote-link", {
```

Comment:
A modal would be slightly obnoxious just for copying. A tooltip or other minor form of feedback like a checkmark icon popping up next to the clicked button, would work better